client/src/Movies/SavedList.js

```diff
@@ -1,18 +1,35 @@
 import React, { Component } from 'react';
+import { Link, NavLink } from 'react-router-dom';
 
 export default class SavedList extends Component {
   constructor(props) {
```

Comment:
I know you didn't write this code, but think about whether or not this component needs to be a class component.